### ISSUE TITLE
Bug: Note that some plugins currently do not support ESLint v9 yet. You may need to use '--force' when installing

### ISSUE THREAD

**Issue by Byron2016** [eslint/eslint#18766]
### Environment

Node version:  v20.15.1
npm version:  v10.8.2
Local ESLint version:  v9.9.0
Global ESLint version:  Not found
Operating System:  win32 10.0.19045

### What parser are you using?

Default (Espree)

### What did you do?

```
pnpm create @eslint/config@latest
```
```
√ How would you like to use ESLint? · problems
√ What type of modules does your project use? · esm
√ Which framework does your project use? · react
√ Does your project use TypeScript? · typescript
√ Where does your code run? · browser
The config that you've selected requires the following dependencies: eslint@9.x, globals, @eslint/js, typescript-eslint, eslint-plugin-react
√ Would you like to install them now? · No / Yes
√ Which package manager do you want to use? · pnpm

```

### What did you expect to happen?

ESLint run its own config file without problems

### What actually happened?

```js
devDependencies:
+ @eslint/js 9.9.0
+ eslint 9.9.0
+ eslint-plugin-react 7.35.0
+ globals 15.9.0
+ typescript-eslint 8.0.1

Done in 42.7s
Successfully created 

**Note that some plugins currently do not support ESLint v9 yet.
You may need to use '--force' when installing**

```

### Link to Minimal Reproducible Example

none

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

none

**Comment by Tanujkanti4441** [comment]
Hi @Byron2016, thanks for the issue, but can you explain what exactly the bug is.

**Comment by Byron2016** [comment]
Hi @Tanujkanti4441 thanks for your assitance. 
The problem is when you run `pnpm create @eslint/config@latest` the tool is giving us these message:

```js
Successfully created 

**Note that some plugins currently do not support ESLint v9 yet.
You may need to use '--force' when installing**
```

Then people (me) don´t know if there are or not a problem (ssuccessfully created / some plugins currently do not suppor ES....) 

I also asked in **eslint-plugin-react** issues page [[Bug]: Support ESLint v9.9.0](https://github.com/jsx-eslint/eslint-plugin-react/issues/3800) and @ljharb answered me 

```js
 yes, the peer deps explicitly say that it supports eslint ^9.7. The warning isn't specific to this plugin, it's from that eslint/config package.
```
 [response-ljharb](https://github.com/jsx-eslint/eslint-plugin-react/issues/3800#issuecomment-2278802568)

What I think could be the right path is to talk **eslint team** and **eslint-plugin-react team** and coordinate some actions to solve this problem.

Then What do you think about to try to have a meeting find the problen and solve it  (if exists) with both teams?

**Comment by ljharb** [comment]
It’s a general message, not specific to any plugin, so i think this is just a case of user confusion. The message can probably be improved.

**Comment by Tanujkanti4441** [comment]
yes, it is just a note not an error message, it's just says there can be plugins that don't have eslint `v9` as there peerDependencies (but have v8) in that case you can use `--force` when installing those plugins (like `eslint-config-airbnb` right now).

**Comment by Byron2016** [comment]
Hi @Tanujkanti4441   @ljharb thnaks for your answer.
My new question: 
1.- If I use --force in a program thant don´t support other program, how can I be sure there are not problems that I will not notice?
2.- I know that this propose it is not happend, but I need to propose: It is necesary to improve message, eslint must have the capacity to analice if dependencies that depents its "own config function" is 100% secure. If it is installing:  this dependenciees: @eslint/js , eslint , eslint-plugin-react, globals, typescript-eslint all of them must be 100% with eslint.

Thanks again for your time. 🙏
I suppose after this this inssue can be closed? But I let you to know my opinion. ✌

**Comment by aladdin-add** [comment]
it should have been fixed by https://github.com/eslint/create-config/pull/141, however it was not released successfully.

we'll do a new release later, please follow the pr for new updates: https://github.com/eslint/create-config/pull/145

**Comment by Byron2016** [comment]
With a merge of [https://github.com/eslint/create-config/pull/145](https://github.com/eslint/create-config/pull/145) the message is not show.

```bash
Note that some plugins currently do not support ESLint v9 yet.
You may need to use '--force' when installing
```
